### DISCUSSION TITLE
Automatic cache clearing / choosing the cache store

### DISCUSSION THREAD

**Discussion by binaryfire** [ash-jc-allen/laravel-exchange-rates#120] [Q&A]
Great package Ash! I’ve got an app with multiple cache stores. Is there any way to specify which store the package should use? 

Also, is there an easy way to clear all cached rates? I’m not using historical rates (just daily rates). I need to get / use the latest rates each day.

**Comment by ash-jc-allen** [comment]
Hey @binaryfire, thanks! Unfortunately, there isn't a way to specify the cache stores sorry. However, if this is something that enough people want, it's something I'd potentially be interested in adding in the future.

I like to use the `shouldBustCache` and `shouldCache` methods when I'm working with live exchange rates for the day. This part in the docs should hopefully help you out: [Cache](https://github.com/ash-jc-allen/laravel-exchange-rates#preventing-exchange-rates-from-being-cached)

Feel free to let me know if this doesn't solve your issue though 🙂
